pkg/nvidia-query/nvml/error.go

```diff
@@ -11,6 +11,16 @@ func IsNotSupportError(ret nvml.Return) bool {
 	if ret == nvml.ERROR_NOT_SUPPORTED {
 		return true
 	}
+
+	// "Argument version mismatch" indicates that the NVML library
+	// is not compatible with the corresponding API call
+	// thus marking the operation as not supported.
+	if ret == nvml.ERROR_ARGUMENT_VERSION_MISMATCH {
+		return true
+	}
```

Comment:
let's not mix them together. Create a function for VersionMisatch, and just call it in GPM-related code.

Reply:
addressed